gcc/testsuite/rust/compile/builtin_macro_include_bytes_location_info.rs

```diff
@@ -0,0 +1,10 @@
+#![feature(rustc_attrs)]
+
+#[rustc_builtin_macro]
+macro_rules! include_bytes {
+    () => {{}};
+}
+
+fn main() {
+    include_bytes!("nonexistent.txt"); // { dg-error "cannot open filename (.*?)nonexistent.txt: No such file or directory" }
```

Comment:
Any reason you needed to add the `(.*?)` to the regex?

Reply:
When running the tests, the name of the file is reported as an absolute path, so this will match any path with the file name at the end